### DISCUSSION TITLE
Get the Attribute Datetime

### DISCUSSION THREAD

**Discussion by Spawtt** [AutomaApp/automa#704] [Q&A] [answered]
Hello everyone, I need your help.
Who can tell me how to get the datetime value in this case  ?

![Datetimepiker](https://user-images.githubusercontent.com/110847843/183483411-81098149-c3b6-46e4-b39e-1c6cd7ec0975.png)

I've tried a lot of things, but I'm stuck.

Thank you in advance.

**Comment by Kholid060** [comment, marked as answer]
Use the [attribute value](https://docs.automa.site/blocks/attribute-value.html#attribute-value-block) block, and input `datetime` in the attribute name field on that block.

**Reply by Spawtt** [reply]
It's ok thanks.
